javascript/javascript1/week2/candy_helper.js

```diff
@@ -0,0 +1,45 @@
+const candyStore = 
+{   "Sweet": 0.5,
+    "Chocolate": 0.7, 
+    "Toffee": 1.1,
+    "Chewing-gum": 0.03
+}
+const boughtCandyPrices = []
+
+function addCandy(candyType, weight){
+    
+    let findWeight = candyStore[candyType]
+    const total = findWeight * weight
+    boughtCandyPrices.push(total);
+    return boughtCandyPrices; 
+    
+
+}
+
+const amountToSpend = Math.random() * 100;
+console.log(amountToSpend)
+
+console.log(addCandy("Sweet", 20))
+console.log(addCandy("Toffee", 10))
+console.log(addCandy("Chewing-gum", 750))
+console.log(addCandy("Chocolate", 30))
+
+function canBuyMoreCandy(){
+    let message = " "
+    let total = 0
+    for(let item in boughtCandyPrices){
```

Comment:
Nice to not make your own indexes. Can you see how `for...of` could improve this? 🔍 

Remember to be consistent with your casing of variable names! `CanIBuyMore` starts with a capitalized C whereas others do not. For variables, I recommend sticking to lowercased initial letters. Many attribute a different meaning to variables starting with a capitalized letter as is evident with GitHub also showing it in a different color! 👀

Reply:
yes easier and shorter 😺 
thanks for pointing it out.

i should check my code more often
there is formatting, variable names, for loops